### PR TITLE
Installed Vuex

### LINKED ISSUE #6: 1.2-Logging into SSO portal as existing user
Team: RED
Complexity: Medium
Time Est. 60hrs 

Update: 14 hrs as of 3/7/19. Maybe 20 by 3/9/19.

### PR DESCRIPTION
**Pull request by cf2080**
# Issue/Ticket

CLOSES #6 

# Description
* Added vuex to client side so certain data can be accessed easier.

## Type of change

List one of the following: Chore

Label your PR accordingly.

# How Has This Been Tested?
testing done manually.

# Notes

Put any notes here

### CODE REVIEW

File: Backend/DataAccessLayer/DataAccessLayer.csproj [hunk 1]
```diff
@@ -75,6 +75,10 @@
     <Compile Include="Migrations\201903111003125_upate-sessions-userNotUnique.Designer.cs">
       <DependentUpon>201903111003125_upate-sessions-userNotUnique.cs</DependentUpon>
     </Compile>
+    <Compile Include="Migrations\201903122358472_pending-migration.cs" />
```
Comment (alfredovargasjr) [inline Backend/DataAccessLayer/DataAccessLayer.csproj:78]: why is there a migration being added? I thought this was just VueX

File: Backend/ManagerLayer/Login/LoginManager.cs [hunk 2]
```diff
@@ -26,11 +26,11 @@ public LoginManager()
             _tokenService = new TokenService();
         }
 
-        public bool LoginCheckUserExists(string email)
```
Comment (alfredovargasjr) [inline Backend/ManagerLayer/Login/LoginManager.cs:29]: Are you also adding changes to the backend?

**Comment by alfredovargasjr** [comment]
Make the Vuex integration separate from your changes to the backend. Make another PR for those backend changes

**Review by alfredovargasjr — CHANGES_REQUESTED** [review]

File: Backend/DataAccessLayer/DataAccessLayer.csproj (hunk 1 above)
Comment (cf2080) [reply Backend/DataAccessLayer/DataAccessLayer.csproj:78]: created a pull request for the front end

**Pull request closed without merging**